### PR TITLE
feat: migrate to use discord-module-loader

### PR DESCRIPTION
**Pull request by EncryptedDev**
Resolves CU-1wgb67e CU-1wgb67e

### CODE REVIEW

**Comment by Timeraa** [comment]
Task linked: [CU-1wgb67e Migrate info command to use autocomplete](https://app.clickup.com/t/1wgb67e)

**Comment by Timeraa** [comment]
Task linked: [CU-1wgb675 Migrate to use the module loader](https://app.clickup.com/t/1wgb675)

File: src/modules/betaAlpha/events/guildMemberUpdate.ts [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+export default new DiscordEvent("guildMemberUpdate", async (oldMember, newMember) => {
 	const oldRoles = oldMember.roles.cache,
 		newRoles = newMember.roles.cache;
 
 	//* Member get's alpha role
 	if (!oldRoles.has(config.roles.alpha) && newRoles.has(config.roles.alpha)) {
 		if (newRoles.has(config.roles.beta)) await newMember.roles.remove(config.roles.beta);
 
-		return await pmdDB
+		return void (await pmdDB
```
Comment (Bas950) [inline src/modules/betaAlpha/events/guildMemberUpdate.ts:15]: return void is no longer needed in latest version

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: src/modules/betaAlpha/events/guildMemberUpdate.ts (hunk 1 above)
Comment (EncryptedDev) [reply src/modules/betaAlpha/events/guildMemberUpdate.ts]: Resolved

**Review by Bas950 — APPROVED** [review]

**Pull request merged**